SlicingDice.cpp

```diff
@@ -1,22 +1,32 @@
 #include "SlicingDice.h"
 
 
-SlicingDice::SlicingDice(const char* apiUserKey) {
-    host = "api.slicingdice.com/v1";
+SlicingDice::SlicingDice(String apiUserKey) {
+    host = "api.slicingdice.com";
     port = 80;
     apiKey = apiUserKey;
+    test = 0;
 }
 
-SlicingDice::SlicingDice(const char* apiUserKey, const char* customHost) {
+SlicingDice::SlicingDice(String apiUserKey, const char* customHost) {
     host = customHost;
```

Comment:
when on repeated construction like these, I refactor the code to a contruct() method